src/i_video.c

```diff
@@ -662,7 +667,6 @@ static void UpdateRender(void)
     SDL_LowerBlit(screenbuffer, &blit_rect, argbbuffer, &blit_rect);
     SDL_UpdateTexture(texture, &blit_rect, argbbuffer->pixels,
                       argbbuffer->pitch);
-    SDL_RenderClear(renderer);
```

Comment:
Without this clear for pillar/letterbox cases stuff like RTSS that draws into the framebuffer will get messed up, and `vga_porch_flash` won't work. Other than that this patch seems fine

Reply:
I fixed vga_porch here: https://github.com/fabiangreffrath/woof/pull/1847/commits/6380fa33c370dad2e321c29ef0600cd6c1b5c83a

What if we just call `SDL_RenderClear(renderer)` unconditionally? Seems safer, the GPU is idle most of the time anyway.

Reply:
It's just useless work that can be avoided with a simple flag check 🤷‍♂️

Reply:
Okay, fixed here: https://github.com/fabiangreffrath/woof/pull/1847/commits/5f63d4cfcbb3f0c674b724ff750902082a461743